go.mod

```diff
@@ -15,6 +16,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.6.0
 )
 
+require github.com/google/go-cmp v0.5.8 // indirect
```

Comment:
why is this require in a separate block ?

Reply:
I have no idea- I didn't add it manually. I'll try merging them.

Reply:
Fixed